npm run github-deploy (dev and prod) fail

**Note: for support questions, please use one of these channels:** [Chat: AngularClass.slack](http://angularclass.com/member-join/) or [Twitter: @AngularClass](https://twitter.com/AngularClass)

* **I'm submitting a ...**
[X] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Do you want to request a *feature* or report a *bug*?**
Reporting a Bug


* **What is the current behavior?**
Errors when using github-deploy

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via


Clone the repository
install NPM packages with npm install
run `npm run github-deploy:prod`

(if you run github-deploy without prod flag, you get tslint-loader errors that I think are red-herrings for the real problem)

Errors will occur while compiling main and polyfills.
The errors indicate that a module is not found at node_modules/typescript/lib/package.json

* **What is the expected behavior?**
Github deploy should not error

* **What is the motivation / use case for changing the behavior?**
it is a bug


* **Please tell us about your environment:**

- Angular version: 2.4.6
- Browser: Not applicable
- OS: Windows 10

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)
```
ERROR in ./src/polyfills.browser.ts
Module build failed: Error: Cannot find module 'C:\_Dev\Github\dbosley-info\node_modules\typescript\lib\package.json'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at setupTs (C:\_Dev\Github\dbosley-info\node_modules\awesome-typescript-loader\src\instance.ts:162:29)
    at Object.ensureInstance (C:\_Dev\Github\dbosley-info\node_modules\awesome-typescript-loader\src\instance.ts:84:24)
    at compiler (C:\_Dev\Github\dbosley-info\node_modules\awesome-typescript-loader\src\index.ts:40:22)
    at Object.loader (C:\_Dev\Github\dbosley-info\node_modules\awesome-typescript-loader\src\index.ts:14:18)

ERROR in ./src/main.browser.ts
Module build failed: Error: Cannot find module 'C:\_Dev\Github\dbosley-info\node_modules\typescript\lib\package.json'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at setupTs (C:\_Dev\Github\dbosley-info\node_modules\awesome-typescript-loader\src\instance.ts:162:29)
    at Object.ensureInstance (C:\_Dev\Github\dbosley-info\node_modules\awesome-typescript-loader\src\instance.ts:84:24)
    at compiler (C:\_Dev\Github\dbosley-info\node_modules\awesome-typescript-loader\src\index.ts:40:22)
    at Object.loader (C:\_Dev\Github\dbosley-info\node_modules\awesome-typescript-loader\src\index.ts:14:18)
```

--- a/config/webpack.github-deploy.js
+++ b/config/webpack.github-deploy.js
@@ -6,8 +6,6 @@ const path = require('path');
 const helpers = require('./helpers');
 const ghDeploy = require('./github-deploy');
 const webpackMerge = require('webpack-merge'); // used to merge webpack configs
-const ghpages = require('gh-pages');
-
 
 /**
  * Webpack Constants
@@ -63,6 +61,7 @@ module.exports = function (options) {
          // this is the fix for now.
          fs.writeFileSync(path.join(webpackConfig.output.path, '.nojekyll'), '');
 
+         const ghpages = require('gh-pages');
          ghpages.publish(webpackConfig.output.path, options, function(err) {
            if (err) {
              console.log('GitHub deployment done. STATUS: ERROR.');

